2do_fe/src/components/TaskDetail/TaskDetail.js

```diff
@@ -4,11 +4,13 @@ import ProgressIcon from "./ProgressIcon.svg";
 import FinishIcon from "./FinishIcon.svg";
 import StageButton from "../Done/StageButton.js";
 import { openEditDesc } from "../PopUps/EditTaskPopUp";
+import RemoveCircleIcon from "@mui/icons-material/RemoveCircle";
 //import { openTaskDesc } from "../AddTask/AddTaskPopUp.js";
 import Moment from "react-moment";
 
```

Comment:
//Add this here

import taskActions from "../../redux/actions/task.actions";
import { useDispatch } from "react-redux";